packages/hub/test/hub.test.ts

```diff
@@ -194,113 +187,104 @@ describe('Hub', () => {
   });
 
   describe('captureException', () => {
+    const mockClient: any = { captureException: jest.fn() };
+    beforeEach(() => {
+      mockClient.captureException.mockClear();
+    });
+
     test('simple', () => {
-      const hub = new Hub();
-      const spy = jest.spyOn(hub as any, '_invokeClient');
+      const hub = new Hub(mockClient);
       hub.captureException('a');
-      expect(spy).toHaveBeenCalled();
-      expect(spy.mock.calls[0][0]).toBe('captureException');
-      expect(spy.mock.calls[0][1]).toBe('a');
+      expect(mockClient.captureException).toHaveBeenCalled();
+      expect(mockClient.captureException.mock.calls[0][0]).toBe('a');
     });
 
     test('should set event_id in hint', () => {
-      const hub = new Hub();
-      const spy = jest.spyOn(hub as any, '_invokeClient');
+      const hub = new Hub(mockClient);
       hub.captureException('a');
-      // @ts-ignore Says mock object is type unknown
-      expect(spy.mock.calls[0][2].event_id).toBeTruthy();
+      expect(mockClient.captureException.mock.calls[0][1].event_id).toBeTruthy();
     });
 
     test('should generate hint if not provided in the call', () => {
-      const hub = new Hub();
-      const spy = jest.spyOn(hub as any, '_invokeClient');
+      const hub = new Hub(mockClient);
       const ex = new Error('foo');
       hub.captureException(ex);
-      // @ts-ignore Says mock object is type unknown
-      expect(spy.mock.calls[0][2].originalException).toBe(ex);
-      // @ts-ignore Says mock object is type unknown
-      expect(spy.mock.calls[0][2].syntheticException).toBeInstanceOf(Error);
-      // @ts-ignore Says mock object is type unknown
-      expect(spy.mock.calls[0][2].syntheticException.message).toBe('Sentry syntheticException');
+      expect(mockClient.captureException.mock.calls[0][1].originalException).toBe(ex);
+      expect(mockClient.captureException.mock.calls[0][1].syntheticException).toBeInstanceOf(Error);
+      expect(mockClient.captureException.mock.calls[0][1].syntheticException.message).toBe('Sentry syntheticException');
```

Comment:
```suggestion
      const hint = mockClient.captureException.mock.calls[0][1];
      expect(hint.originalException).toBe(ex);
      expect(hint.syntheticException).toBeInstanceOf(Error);
      expect(hint.syntheticException.message).toBe('Sentry syntheticException');
```

Here and elsewhere - maybe a little easier to read this way? Or since there are so many of them, make a `getCaptureExceptionArgs(mockClient)`? Your call.